tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "target": "es3",
```

Comment:
This module is cosidered to use with Node.js. So I want to suggest `"es5"` as a target.